src/api/operator/utils.ts

```diff
@@ -68,6 +68,28 @@ export function getOrderFilledAmount(order: RawOrder): { amount: BigNumber; perc
   return { amount: executedAmount, percentage: executedAmount.div(totalAmount) }
 }
 
+export function getSurplus(
```

Comment:
this function is pure, why here in the "operator" and not in utils?

Reply:
They are still strongly related. I'm using a single test file for both, for instance.

But ok, I give up =P
Will migrate this utils out of the api into regular utils